lib/clean-evm-node.js

```diff
@@ -16,6 +16,21 @@ const buildDisplayInfo = k => {
     stack_kast
   } = getInfo(k);
 
+  let kcell = kast.get(k, 'k');
+
+  const reduceKSequence = term => {
+    if(term.node == "KApply" && term.label == "#KSequence") {
+      return [term.args[0]].concat(reduceKSequence(term.args[1]))
+    } else {
+      return [term];
+    }
+  }
+  const findCodeDeposit = reduceKSequence(kcell)
+    .find(term => term.node == "KApply" && term.label == "#codeDeposit__EVM")
+  const codeDeposit = findCodeDeposit
+    && kast.format(findCodeDeposit.args[0]);
+  const isCreate = call_id == codeDeposit;
```

Comment:
Not a JS expert, but could `===` be better here? Cf. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators#Identity

Reply:
think in this case it doesn't matter much

Reply:
think in this case it doesn't matter much

Reply:
think in this case it doesn't matter much